client/src/atoms/ProfiAva.jsx

```diff
@@ -0,0 +1,89 @@
+                        </g>
+                        <path d="M33.5 13.5208C35.782 13.5208 37.9705 14.4273 39.5841 16.0409C41.1977 17.6545 42.1042 19.843 42.1042 22.125C42.1042 24.4069 41.1977 26.5954 39.5841 28.209C37.9705 29.8226 35.782 30.7291 33.5 30.7291C31.218 30.7291 29.0295 29.8226 27.4159 28.209C25.8023 26.5954 24.8958 24.4069 24.8958 22.125C24.8958 19.843 25.8023 17.6545 27.4159 16.0409C29.0295 14.4273 31.218 13.5208 33.5 13.5208ZM16.2917 19.6666C17.6683 19.6666 18.9467 20.0354 20.0529 20.6991C19.6842 24.2146 20.7167 27.7054 22.8308 30.4341C21.6017 32.7941 19.1433 34.4166 16.2917 34.4166C14.3357 34.4166 12.4598 33.6396 11.0768 32.2566C9.69367 30.8735 8.91667 28.9976 8.91667 27.0416C8.91667 25.0857 9.69367 23.2098 11.0768 21.8267C12.4598 20.4437 14.3357 19.6666 16.2917 19.6666ZM50.7083 19.6666C52.6643 19.6666 54.5402 20.4437 55.9232 21.8267C57.3063 23.2098 58.0833 25.0857 58.0833 27.0416C58.0833 28.9976 57.3063 30.8735 55.9232 32.2566C54.5402 33.6396 52.6643 34.4166 50.7083 34.4166C47.8567 34.4166 45.3983 32.7941 44.1692 30.4341C46.3125 27.6671 47.3074 24.1806 46.9471 20.6991C48.0533 20.0354 49.3317 19.6666 50.7083 19.6666ZM17.5208 44.8646C17.5208 39.7758 24.6746 35.6458 33.5 35.6458C42.3254 35.6458 49.4792 39.7758 49.4792 44.8646V49.1666H17.5208V44.8646ZM4 49.1666V45.4791C4 42.0621 8.64625 39.1858 14.9396 38.35C13.4892 40.0216 12.6042 42.3325 12.6042 44.8646V49.1666H4ZM63 49.1666H54.3958V44.8646C54.3958 42.3325 53.5108 40.0216 52.0604 38.35C58.3537 39.1858 63 42.0621 63 45.4791V49.1666Z" fill="#F1AC8E" />
+                        <defs>
```

Comment:
svgs should be in asset/icons folder. And there should be `index.js` file exporting those icons.